pages/md-style-guide.md

```diff
@@ -1,191 +1,165 @@
-```
+Try not to be confused by the two uses of the word "object" here:
+Ruby uses the word `Object` to mean the generic base class for
+everything, whereas in JSON it is used only to mean a mapping from
+string keys to values.
 
-### Example
+[tab "Objective-C"]
+In Objective-C, "objects" are analogous to the `NSDictionary` type.
 
-<Warning label="label"> message to show. </Warning>
+[tab "Swift"]
+In Swift, "objects" are analogous to the `Dictionary` type.
+[tabs-end]
 
----
+### Star tag
 
-## Tip
+Highlight changes introduced by drafts with the start tag.
 
-Tip is a special type of info box in which text showing a tip can be shown. The text written between these tags will be shown.
+`<Star label="New in draft"/>`
 
-```markdown
-<Tip label="label"> message to show. </Tip>
-```
+Example:
 
-### Example
+<Star label= "New in draft 2019-09"/>
 
-<Tip label="label"> message to show. </Tip>
+The deprecated keyword is a boolean that indicates that the instance value the keyword applies to should not be used and may be removed in the future.
 
----
+### Infobox tag
 
-## Danger
+Use the infobox tag for [Notes](https://developers.google.com/style/notices). Notes provide the reader with relevant information that is not necessary, for example, an aside or a tip. 
 
-Danger is a special type of info box in which text showing any danger can be shown. The text written between these tags will be shown.
+`<Infobox label="Note"> Message to be shown </Infobox>`
 
-```markdown
-<Danger label="label"> message to show. </Danger>
-```
+Example:
 
-### Example
+<Infobox label="Note"> Draft-07 core and validation are backwards-compatible with draft-06. </Infobox>
 
-<Danger label="label"> message to show. </Danger>
+### Warning tag
 
----
+Use the warning tag for cautions. A [Caution](https://developers.google.com/style/notices) notice lets the reader know they should proceed carefully. 
 
-## Table of Contents
+`<Warning label="Caution"> Message to be shown </Warning>`
 
-Table of Contents provides a structured overview of the main sections or chapters within the current page/document.
+Example:
 
-```markdown
-<tableofcontent content={content} depth={depth} />
-```
+<Infobox label="Caution"> Using `oneOf` increases processing times because every sub-schema requires verification. Prefer `anyOf` whenever possible.<Infobox>
 
----
+### Danger tag
 
-## Tabs Group
+Use the danger tag for [Warning](https://developers.google.com/style/notices) notices, which are stronger than Cautions.  
 
-This can be used to group multiple tabs under a common name and differentiated by specific labels and content.
+`<Danger label="Warning"> Message to be shown </Danger>`
 
-```markdown
-[ tabs-start "label" ]
-[ tab "name" ]
-message to show
-[ tabs-end ]
-```
+Example:
 
-### Example
+<Danger label="Warning"> Modifying a JSON Schema without validating the existing data against it, can cause system or application errors or compromise data integrity. <Infobox>
```

Comment:
The <Danger> tag is closed with <Infobox> and hence not working as expected. Needs to be closed with </danger>